dd-sdk-android-gradle-plugin/src/main/kotlin/com/datadog/gradle/plugin/DdCheckSdkDepsTask.kt

```diff
@@ -68,6 +101,8 @@ abstract class DdCheckSdkDepsTask :
                     )
                 }
             }
+        } else {
+            isLastRunSuccessful = true
```

Comment:
What happens if the task succeeded and in the meantime I change my dependencies and remove Datadog dependency from there. This task would still be considered as `up-to-date` not ?

Reply:
Yes, I mentioned it in the PR description. This is an edge case which cannot be solved with current inputs.